{"name": "Geschichte_(Erzgebirge)", "id": 6203406, "predecessors": [{"name": "Europäische_Geschichte", "id": 235327}, {"name": "!Hauptkategorie", "id": 235489}], "successors": [{"name": "Wüstung_im_Erzgebirge", "id": 6201176}]}